src/views/Expenses.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
+  const [expenses, setExpenses] = useState([]);
+
+  // Run on initial render only
+  useEffect(() => {
+    // Fetch expense items
+    const token = process.env.REACT_APP_MOSTASH_API_KEY;
+    fetch(`http://pi.motine.de:12305/items.json?stash=${token}&kind=expense`)
+      .then(response => response.json())
+      .then(expenseData => setExpenses(expenseData));
+  }, []);
+
+  const addExpense = (newExpense) => {
+    const requestOptions = {
+      method: 'POST',
+      headers: {
+        'Stash': process.env.REACT_APP_MOSTASH_API_KEY,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(newExpense)
+    };
+
+    fetch('http://pi.motine.de:12305/items.json?kind=expense', requestOptions)
+      .then(response => response.json())
+      .then(newExpenseData => setExpenses([...expenses, newExpenseData]));
+  }
+
+  const removeExpense = (id) => {
+    const requestOptions = {
+      method: 'DELETE',
+      headers: {
+        'Stash': process.env.REACT_APP_MOSTASH_API_KEY,
```

Comment:
maybe we can have a constant/function which generates the general headers (to avoid repetition/improve readability)?

Reply:
Sure thing, great idea!